doc/user/content/get-started/isolation-level.md

```diff
@@ -106,11 +106,58 @@ of the rows that were seen by T1.
-and [strengthening correctness](https://github.com/MaterializeInc/materialize/issues/13107) for more details. If some
+next read transaction. See [real-time recency](#real-time-recency) and [strengthening correctness](https://github.com/MaterializeInc/materialize/issues/13107) for more details. If some
 piece of data has been fully ingested from an upstream source AND is included in the results of some read transaction
 THEN all subsequent read transactions are guaranteed to see that piece of data.
 
+### Real-time recency
+
+{{< private-preview />}}
+
+Materialize offers a form of "cross-system linearizability" known as real-time
+recency. When using real-time recency, all client-issued `SELECT` statements
+include at least all data visible to Materialize in any external source (i.e.,
+sources created with `CREATE SOURCE` that use `CONNECTION`s, such as Kafka,
+MySQL, and PostgreSQL sources) shortly after Materialize receives the query.
+This is what we mean by linearizable––the results are guaranteed to contain all
+visible data according to physical time.
+
+For example, real-time recency ensures that if you have just performed an
+`INSERT` into a PostgreSQL database that Materialize ingests as a source, all of
+your real-time recency queries will include the just-written data in their
+results.
+
+Note, however, that we only guarantee the results will contain the data visible
+to Materialize. We do not guarantee that we will consider _only_ the data
+visible at the time we receive the query. For instance, the rate at which
+Materialize ingests data might include additional data made visible after the
+timestamp we determined to be "real-time recent." Another example is that, due
+to network latency, the timestamp from the external system that we determine to
+be "real-time recent" might be later than expected.
+
+Because Materialize waits until it ingests the data from the external system,
+real-time recency queries can have additional latency. This latency is
+introduced by both the time it takes us to ingest and commit data from the
+source and the time spent communicating with it to determine what data it has
+made available to us (e.g., querying PostgreSQL for the replication slot's LSN).
+
+**Details**
+
+- Enable this feature per session using `SET REAL_TIME_RECENCY TO TRUE`.
+- Control the timeout for connecting to the external source to determine the
+  timestamp with the `REAL_TIME_RECENCY_TIMEOUT` session variable.
+- Real-time recency queries only guarantee visibility of data from external
+  systems (e.g., sources like Kafka, MySQL, and PostgreSQL). Real-time recency
+  queries do not offer any form of guarantee when querying Materialize-local
+  objects, such as LOAD GENERATOR sources or system tables.
+- Each real-time recency query connects to each external source transitively
+  referenced in the query. The more external sources that are referenced, the
+  greater the likelihood of network-induced latency.
```

Comment:
```suggestion
- Each real-time recency query connects to each external source transitively
  referenced. This means that the latency overhead will be proportional to the
  number of sources referenced in the query.
```

Reply:
These queries occur in parallel (or at least concurrently) so the latency overhead is not proportional to the number of items queried, but the latency _is_ the max of all of the requests. Not sure if there's a better way to make that clear.